docs/development/setup-vagrant.md

```diff
@@ -1049,7 +1049,7 @@ for the IP address that means any IP address can connect to your development ser
 [vagrant-dl-deb]: https://releases.hashicorp.com/vagrant/2.0.2/vagrant_2.0.2_x86_64.deb
 [vagrant-lxc]: https://github.com/fgrehm/vagrant-lxc
 [vbox-dl]: https://www.virtualbox.org/wiki/Downloads
-[vbox-dl-macos]: https://download.virtualbox.org/virtualbox/5.2.6/VirtualBox-5.2.6-120293-OSX.dmg
+[vbox-dl-macos]: https://download.virtualbox.org/virtualbox/5.2.20/VirtualBox-5.2.20-125813-OSX.dmg
```

Comment:
Looka like you missed a reference:

```
$ git grep 5[.]2[.]6
docs/development/setup-vagrant.md:2. Install [VirtualBox][vbox-dl-macos] (5.2.6).
docs/development/setup-vagrant.md:2. Install [VirtualBox][vbox-dl] (version == 5.2.6).
docs/development/setup-vagrant.md:[vbox-dl-macos]: https://download.virtualbox.org/virtualbox/5.2.6/VirtualBox-5.2.6-120293-OSX.dmg
```

Reply:
![screenshot 2018-11-03 at 3 34 35 am](https://user-images.githubusercontent.com/39343253/47942863-8d252b80-df19-11e8-9465-eb6545542c50.png)
I am not able to edit this file without jumping to master branch! Should I do it by jumping on master branch and submit another PR, or is there any way to edit this file on this same branch?